src/firefly/js/visualize/ui/GatorProtocolRootPanel.jsx

```diff
@@ -0,0 +1,202 @@
+            {lockTitle && lockService &&
+                <Typography {...{level:'title-lg', color:'primary', sx:{width:'17rem', mr:1, mb:1} }}>
+                    {lockTitle}
+                </Typography>
+            }
+            { showWarning ? <ServiceWarning {...{error:false,serviceUrl,title}}/> :
+                <IrsaCatalogSearch serviceUrl={serviceUrl}
+                                   title= {serviceUrl ? title : 'IRSA'}
+                                   showSqlSection={getGatorProtoService(serviceUrl)?.showSqlSection} />
+            }
+        </Stack>
+    );
+}
+
+function ServiceWarning({error,serviceUrl, title}) {
+    if (!serviceUrl) {
+        return (
+            <Typography level='h3' color='warning' sx={{textAlign:'center', mt:5}}>
+                {`Select a ${title} Service`}
+            </Typography>
+        );
+    }
+    else if (error)  {
+        return (
+            <Stack>
+                <Typography level='h3' color='warning' sx={{textAlign:'center', mt:5}}>Error</Typography>
+                <Typography color='warning' sx={{textAlign:'center', mt:5}}>{error}</Typography>
+            </Stack>
+        );
+    }
+    return <div/>;
+}
+
+function Services({serviceUrl, servicesShowing, gpOps, onGatorProtoServiceOptionSelect,title} ) {
+    const [extraStyle,setExtraStyle] = useState({overflow:'hidden'});
+    const enterUrl=  useFieldGroupValue('enterUrl')[0]();
+
+    useEffect(() => {
+        if (servicesShowing) {
+            setTimeout(() => setExtraStyle({overflow:'visible'}), 250);
+        }
+        else {
+            setExtraStyle({overflow:'hidden'});
+        }
+    }, [servicesShowing]);
+
+    return (
+        <Stack sx={{
+            height: servicesShowing ? 'auto' : 0,
+            pb: servicesShowing ? 1.5 : 0,
+            justifyContent:'space-between',
+            alignItems:'center',
+            transition: 'all .2s ease-in-out', //to animate height changes (hide/show Services)
+            ...extraStyle}}>
+            <Stack direction='row' spacing={1} sx={{alignItems:'center', width:1}}>
+                <Stack alignItems='flex-start' spacing={1}>
+                    <Typography {...{level:'title-lg', color:'primary', sx:{width:'17rem', mr:1} }}>
+                        Select Source
+                    </Typography>
+                    <SwitchInputField {...{ size:'sm', endDecorator:'Enter my URL', fieldKey:'enterUrl',
+                        initState:{value:false},
+                        sx: {
+                            alignSelf: 'flex-start',
+                            '--Switch-trackWidth': '20px',
+                            '--Switch-trackHeight': '12px',
+                        },
+                    }} />
+                </Stack>
+                <Stack>
+                    {enterUrl ? (
+                        <InputField orientation='horizontal'
+                                    placeholder='Enter SIAv2 Url'
```

Comment:
```suggestion
                                    placeholder='Enter IPAC Gator protocol URL'
```

Or you could use the approach at line 156 where there's a variable like `title` that is used to state the type of protocol?

Reply:
I will do that.